packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -428,9 +432,70 @@ export default {
           },
         },
       });
+      this.insertContent();
       this.addEditorListeners();
     },
 
+    /**
+     * This function is necessary as tiptap doesn't render the content passed
+     * directly through `editor.commands.setContent` the content passed down to it
+     * should be already parsed. So We're parsing the elements into it's corresponding
+     * HTML version before setting it.
+     */
```

Comment:
I wonder if this would work better if we used json instead of html. Anyways doesn't matter for now, it's not an option due to how our backend currently works.

Reply:
I think it would work exactly the same, why do you think it'd work better with json? 🤔